<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="uploadButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>561</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>29</y>
      <width>561</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="threshButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>430</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary Thresholding</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>561</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>396</y>
      <width>551</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="sliderLow">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>440</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderHigh">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>440</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="gaussianButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian Blur</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderKernel">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>480</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="grayButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>390</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Scale</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderMedian">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>520</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="medianButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>510</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Median Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="histogramButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>550</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dilationButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>590</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dilation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="erosionButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>630</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Erosion</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderDilation">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>600</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderErosion">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>640</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chainModeCheckBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Chain Mode</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="operationsTextEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1133</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
